### DISCUSSION TITLE
Option to add a shortcut to a subreddit to the home screen

### DISCUSSION THREAD

**Discussion by vishalnandagopal** [Docile-Alligator/Infinity-For-Reddit#982] [Ideas]
![Screenshot_2022-05-13-11-51-19-310_ml.docilealligator.infinityforreddit.jpg](https://user-images.githubusercontent.com/61023404/168223206-4ac0dde5-fb1f-4882-beb8-09ecf5da135a.jpg)

An option could be added to this menu to make it easier to open a specific subreddit. Clicking that option would add a shortcut to homescreen, with the logo of the sub as the shortcut icon.

The current way of opening a subreddit is
1. Open infinity
2. Search subreddit name in search bar
3. Charge to subreddit tab
4. Find subreddit and click on it
which I feel is too long if you frequently open subreddits.

**Comment by acolombo11** [comment]
I had the same idea and found this comment. I will take a look at the contributions guidelines and try to implement this feature, it should be rather quick

**Comment by acolombo11** [comment]
I opened this [PR#1485](https://github.com/Docile-Alligator/Infinity-For-Reddit/pull/1485)
